docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+services:
+  dev:
+    container_name: core-dev
+    user: vscode
+    build:
+      context: .
+      dockerfile: docker/Dockerfile.core.dev
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - .:/workspace:cached
+    command: /bin/sh -c "while sleep 1000; do :; done"
+  frigate:
+    container_name: frigate
+    privileged: true
+    build:
+      context: .
+      dockerfile: docker/Dockerfile.amd64
+      # dockerfile: docker/Dockerfile.core.dev
+    devices:
+      - /dev/bus/usb:/dev/bus/usb
+      - /dev/dri:/dev/dri # for intel hwaccel, needs to be updated for your hardware
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ./config/config.yml:/config/config.yml:ro
```

Comment:
This line is wrong imho, or the docu is not clear to me
With this it expects a folder "config/config.yml" not a YAML file
My workaround is:

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -22,7 +22,7 @@ services:
       - /dev/dri:/dev/dri # for intel hwaccel, needs to be updated for your hardware
     volumes:
       - /etc/localtime:/etc/localtime:ro
-      - ./config/config.yml:/config/config.yml:ro
+      - ./config/config.yml/config.yml:/config/config.yml:ro
       - ./debug:/media/frigate
       - ./frigate:/opt/frigate/frigate:cached
       - ./migrations:/opt/frigate/migrations:cached

and adding the config.yml in the folder config/config.yaml/